### ISSUE TITLE
[BUG/ISSUE] Issue with ALBD (albedo) over polar night and its use to determine ocean/land/ice grid boxes in HEMCO and GEOS-Chem

### ISSUE THREAD

**Issue by jaegle** [geoschem/geos-chem#778]
HEMCO uses the function HCO_LandType to identify whether a grid box is water, land or ice. This is based on the land water index (LWI) and the albedo (ALBD):
```F90   
    ! Set threshold for albedo over ice. All surfaces w/ an albedo above
    ! this value will be classified as ice.
    REAL(sp), PARAMETER :: albd_ice = 0.695_sp

    !--------------------------
    ! HCO_LANDTYPE begins here
    !--------------------------

    ! Water:
    IF ( NINT(WLI) == 0 .AND. Albedo < albd_ice ) THEN
       LandType = 0

    ! Land:
    ELSEIF ( NINT(WLI) == 1 .AND. Albedo < albd_ice ) THEN
       LandType = 1

    ! Ice:
    ELSEIF ( NINT(WLI) == 2 .OR. Albedo >= albd_ice ) THEN
       LandType = 2
    ENDIF
```

Similarly, other parts of the code (calc_met_mod.F90 to determine IsWater, IsIce, IsLand, IsSnow; or hcox_gc_POPs_mod.F90, hcox_soilnox.F90, etc...) use the ALBD variable with thresholds of 0.4 for snow or 0.695 for sea ice. However, the MERRA-2 ALBD variable has values of >0.7 over polar night. I assume that this comes from the fact that the original file (tavg1_2d_rad_Nx) reports no albedo during nighttime and a fill value is used over polar night regions in the file processing. The figure below shows the MERRA-2 0.5x0.625 A1 fields for ALBD and FRSEAICE on 2010-01-01
![2010-01-01](https://user-images.githubusercontent.com/61677108/124995368-1d399e00-dffc-11eb-9292-12e6d43d5c86.png)

So on that day any land or water gridbox poleward of ~67N is assumed to be covered with ice. While is this probably correct for land, this is not the case for parts of the North Atlantic Ocean that have no sea ice.

One fix would be either re-calculate the ALBD variable in the MERRA-2 input files to make use of FRSEAICE over the ocean and some other variables over land (such as FRSNO=fractional area of land snowcover; and FRLANDICE=fraction of land ice) to account for snow/ice. Another wayt to fix this issue would be to replace the use of ALBD throughout the GEOS-Chem and HEMCO code to use a sea ice fraction (FRSEAICE) threshold and FRSNO/FRLANDICE threshold to determine whether ocean and land boxes are covered with sea ice or snow.

This bug leads to the erroneous inhibition of sea salt emissions and any air-sea exchange over ocean grid boxes that are in polar night. This could also affect other parts of the code (emissions, deposition) that rely on IsLand/IsWater/IsIce, or HCO_LandType.

**Comment by yantosca** [comment]
Thanks @jaegle for writing.  Also tagging @sdeastham here.  I think for GCHP's sake we wanted to try to get away from using albedo as a proxy for land cover, since the ALBEDO field from the met fields = 0 at night. 

I think the albedo was used in the land cover back when LWI had different definitions back in the old GEOS-3 or GEOS-4 met in order to get consistency.  But I think we can pretty much drop that now.

**Comment by sdeastham** [comment]
My 2 cents - I 100% agree that we should drop the use of albedo as a proxy for ice cover. Since we always have "real" ice cover data, we should just use that. This would also help to fix the problem where we are using _different_ thresholds in different parts of the code..

**Comment by yantosca** [comment]
This should be resolved by the following PR's:

  - #945
  - https://github.com/geoschem/GCHP/pull
  - [https://github.com/geoschem/HEMCO/pull/945

**Comment by yantosca** [comment]
All of the above PR's have been merged into the 13.4.0 development stream.  We can now close this issue.
